CMake/External_Ceres.cmake

```diff
@@ -26,6 +26,13 @@ else()
   list(APPEND Ceres_EXTRA_BUILD_FLAGS -DMINIGLOG:BOOL=ON)
 endif()
 
+if (fletch_ENABLE_GFlags)
+  list(APPEND Ceres_DEPENDS GFlags)
+  list(APPEND Ceres_EXTRA_BUILD_FLAGS
+         -Dgflags_DIR:PATH=${fletch_BUILD_INSTALL_PREFIX}/lib/cmake/gflags
```

Comment:
Do you mean GFlags_ROOT? It doesn't look like we're using gflags_DIR anywhere else, and GFlags_ROOT is unused by ceres.